deepchecks/core/suite.py

```diff
@@ -143,24 +173,27 @@ def save_as_html(
             whether to use ipywidgets or not
         requirejs: bool , default: True
             whether to include requirejs library into output HTML or not
-        """
-        output_id = get_random_string()
 
+        Returns
+        -------
+        Optional[str] :
+            name of newly create file
+        """
         if file is None:
             file = 'output.html'
         if isinstance(file, str):
             file = create_new_file_name(file)
 
         if as_widget is True:
             widget_to_html(
-                widget=self.to_widget(unique_id=output_id),
+                widget=self.to_widget(unique_id=unique_id or get_random_string(n=25)),
```

Comment:
why the change?

Reply:
previously `output_id` was declared above `if file is None:` now I just replaced it with one statement/

it means that if a user did not pass a `unique_id` we need to generate it by ourselves in order to make sure that navigation links will be rendered